apps/web/components/booking/BookingListItem.tsx

```diff
@@ -145,6 +145,7 @@ function BookingListItem(booking: BookingItemProps) {
   const isRecurring = booking.recurringEventId !== null;
   const isTabRecurring = booking.listingStatus === "recurring";
   const isTabUnconfirmed = booking.listingStatus === "unconfirmed";
+  const isBookingFromRoutingForm = isBookingReroutable(parsedBooking);
```

Comment:
We were calling `isBookingReroutable` in different parts of the component. Refactored so we only have to call it once.